src/Bridges/Constraint/vectorize.jl

```diff
@@ -3,55 +3,144 @@
 
 Transforms a constraint `G`-in-`scalar_set_type(S, T)` where
 `S <: VectorLinearSet` to `F`-in-`S`.
+
+## Examples
+
+The constraint `SingleVariable -in- LessThan{Float64}` becomes
+`VectorAffineFunction{Float64} -in- Nonpositives`, where `T = Float64`,
```

Comment:
I usually write `F`-in-`S` but you write `F -in- S`. Should it be a point for the JuMP style guide ?